driver/src/price_finding/naive_solver.rs

```diff
@@ -110,19 +113,19 @@ impl PriceFinding for NaiveSolver {
         let mut prices: Vec<u128> = vec![0; TOKENS as usize];
         let mut exec_buy_amount: Vec<u128> = vec![0; orders.len()];
         let mut exec_sell_amount: Vec<u128> = vec![0; orders.len()];
-        let mut total_objective_value = U256::zero();
 
         let mut found_flag = false;
 
         for (i, x) in orders.iter().enumerate() {
             for j in i + 1..orders.len() {
                 // Preprocess order to leave "space" for fee to be taken
-                let x = order_with_buffer_for_fee(&x, &self.fee);
+                let x = order_with_buffer_for_fee(x, &self.fee);
```

Comment:
Any particular reason for this change?

Reply:
Nope, reverting. I think it happened when I was making changes to the naive solver and I left it in by accident.